### PR TITLE
Add info about new auth storage class usage.

### PR DESCRIPTION
**Pull request by ADmad**
Refs: https://github.com/cakephp/cakephp/pull/6664

### CODE REVIEW

**Comment by ravage84** [comment]
Looks good.

File: en/controllers/components/authentication.rst [hunk 1]
```diff
@@ -227,14 +228,17 @@ Using Digest and Basic Authentication for Logging In
...
-read user info from session. You may also want to set config
-``unauthorizedRedirect`` to ``false`` which will cause AuthComponent to throw
-a ``ForbiddenException`` instead of default behavior of redirecting to referrer.
-Stateless authentication will re-verify the user's credentials on each request,
-this creates a small amount of additional overhead, but allows clients to
-login in without using cookies and makes is suitable for APIs.
+require a login action in your controller. Stateless authentication will
+re-verify the user's credentials on each request, this creates a small amount of
+additional overhead, but allows clients to login in without using cookies and
+makes is suitable for APIs.
```
Comment (markstory) [inline en/controllers/components/authentication.rst]: is should be AuthComponent here.

Comment (ADmad) [inline en/controllers/components/authentication.rst]: I don't understand.

Comment (markstory) [inline en/controllers/components/authentication.rst]: I fixed it :smile:

**Pull request merged**